example.c

```diff
@@ -0,0 +1,79 @@
+// Includes are not automatically sorted
+#include <a.h> // Comments
+#include "c.h" // Are
+#include <b.h> // ALigned
+
+// declarations:
+
+int SimpleFunction(int a, int b, int c);
+
+void MoreParameters(
+    long long one_number, const char *one_string, const void *some_data);
+
+void ManyParameters(
+    long long one_number,
+    const char *one_string,
+    const void *some_data,
+    const void *more_data);
+
+// definitions:
+
+int SimpleFunction(int a, int b, int c) { return a + b + c; }
+
+int MoreParameters(
+    long long one_number, const char *one_string, const void *some_data)
+{
+    return (int)one_number + (int)one_string + (int)some_data;
+}
+
+int ManyParameters(
+    const long long one_number,
+    const char *const one_string,
+    ARG_UNUSED const void *const some_data,
+    ARG_UNUSED const void *const more_data);
+{
+    return (int)one_number + (int)one_string;
+}
+
+int main(int argc, char **argv)
+{
+    // Most of the time function calls and assignments should fit on one line:
+    int a_number = SimpleFunction(1, 2, 3);
+
+    // Assignments are aligned:
+    int a   = 0;
+    int abc = 10;
```

Comment:
I don't like this. And with multiple assignments, in particular if some are also long ("multi-line function calls"), this results in a code that looks like scattered words to me and is very hard to read for me.

Reply:
You can get around this by putting a newline in front of the long function calls ones.

Reply:
Yeah, I just don't want to be forced to do that. But then again, I'm probably not going to use this tool so I'm fine with this. I will just object to the way it mangles things if you forget to add that extra line.

Reply:
This one's not a big deal for me either way. Let's hear what @craigcomstock thinks about it.

Reply:
@vpodzime I'll turn this off since I agree. I wish it was possible to set some limit for how far it would align something, but this feature doesn't exist (yet).

Reply:
Thanks